index.bs

```diff
@@ -231,6 +231,9 @@ interface NeuralNetworkContext {
   // Create an Operand object that represents a model constant.
   Operand constant(OperandDescriptor desc, ArrayBufferView value);
 
+  // Create a single-value tensor of the specified type from the specified value.
+  Operand constant(float value, optional OperandType type = "tensor-float32");
```

Comment:
The default value of `type` could be `"float32"`. 

If the `type` is specified as `"int32"`, the `float` type of `value` would not match the `type` parameter.

To create a float32 or int32 scalar tensor, it can be implemented in JavaScript as a helper:
```js
function createFloat32Scalar(value) {
  return nn.constant({type: "float32"}, new Float32Array([value]));
}

function createInt32Scalar(value) {
  return nn.constant({type: "int32"}, new Int32Array([value]));
}
```

Reply:
I just think that webidl does allows overloads, and that having to allocate an array of 1 element is unnecessary overhead. Existing frameworks allow creating either a scalar tensor or a constant tensor from a single value just fine. And I would like to avoid requiring the dev to have to provide their own helper function on top of the API if we could make it easier for them. 

For my choice, I went with the most obvious way that works with the current way the `OperandDescriptor` is defined, which I think can still be improved e.g. I'm not entirely clear on why there is a `"float32"` and a `"tensor-float32"` at the same time. A cleaner way could be the type-bound overloads e.g.  `nn.constant(float value)', 'nn.constant(int value)', etc. But that would require that we remove one of the two data types and make data type "float32" to just mean either float value or float tensor. I'm still open to suggestion on this.

Reply:
I prefer that the returned Operand's type be determined by the type of the input: e.g., `nn.constant(float value)', 'nn.constant(int value)', as you suggest. I don't follow why this requires changing "OperandType". Whether we should have both "float32" and "tensor-float32" seems a separate discussion to me (with pros/cons both sides; I prefer having the two types).

Reply:
According to [the restriction of WebIDL overloading](https://heycam.github.io/webidl/#idl-overloading), the numeric types, including `float` and `long`, are not distinguishable. So I am afraid the idea that defining `nn.constant(float value)` and `nn.constant(long value)` would not work.

For scalar constant, based on @wchao1115 's proposal, we may change the `float` to `double` and set default type as `"float32"`. 
```webidl
Operand constant(double value, optional OperandType type = "float32");
```

With that, the code to create a float32 scalar would be:
```js
nn.constant(alpha);
```

To create an int32 scalar:
```js
nn.constant(beta, "int32");
```

It can overload the tensor constant method, as  `double` and `dictionary` are distinguishable.
```
Operand constant(OperandDescriptor desc, ArrayBufferView value);
```

Reply:
Thanks @huningxin. I've updated it per your suggestion. Please take a look at my new commit.

Reply:
It looks good. Thanks!